app/assets/stylesheets/views/manuals.scss

```diff
@@ -59,3 +59,11 @@
   overflow: auto;
   white-space: pre-wrap;
 }
+
+.govuk-body-link {
+  a,
+  a:link,
+  a:visited {
+    color: #0b0c0c;
+  }
+}
```

Comment:
Is there a reason why the default link colour is being overridden?

This doesn't seem like a good idea from a UI consistency point of view.